skrf/media/circularWaveguide.py

```diff
@@ -0,0 +1,433 @@
+'''
+.. module:: skrf.media.rectangularWaveguide
+================================================================
+rectangularWaveguide (:mod:`skrf.media.rectangularWaveguide`)
+================================================================
+
+Represents a single mode of a homogeneously filled circular
+waveguide of cross-section `r^2 pi`. The mode is determined by
+`mode-type` (`'te'` or `'tm'`) and mode indices ( `m` and `n` ).
+
```

Comment:
Unless you plan to take them into account, corrugated circular waveguides can support HE modes as well. So maybe you should indicate it only concerns circular waveguides (not corrugated).